tests/CMakeLists.txt

```diff
@@ -56,9 +56,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/eosd_run_test.py ${CMAKE_CURRENT_BINA
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/eosd_run_remote_test.py ${CMAKE_CURRENT_BINARY_DIR}/eosd_run_remote_test.py COPYONLY)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/consensusValidationMaliciousProducers.py ${CMAKE_CURRENT_BINARY_DIR}/consensusValidationMaliciousProducers.py COPYONLY)
 
+add_test(chain_test chain_test)
+add_test(slow_test slow_test)
 if(UNIX AND NOT APPLE)
-  add_test(chain_test chain_test)
-  add_test(slow_test slow_test)
   add_test(api_test api_test)
```

Comment:
Instead of removing api_test from ubuntu, can you just disable the one failing testcase in api_test and let it run for both ubuntu and mac.